File: javascript/001/main.js
/*
 * ProjectEuler 001
 * JavaScript / main.js
 * copyright (c) 2015 Susisu
 *
 * Node.js v0.12.0
 * `node main.js`
 */

function main() {
    var sum = 0;
    for (var n = 1; n < 1000; n++) {
        if (n % 3 === 0 || n % 5 === 0) {
            sum += n;
        }
    }
    console.log(sum);
}

main();

File: julia/001/main.jl
# ProjectEuler 001
# Julia / main.jl
# copyright (c) 2015 Susisu
#
# Julia v0.4.1
# `julia main.jl`

function main()
    sum = 0
    for n = 1:999
        if n % 3 == 0 || n % 5 == 0
            sum += n
        end
    end
    println(sum)
end

main()
